[2017-05-22 15:09:45.520] [ERROR] logErr - 执行sql发生错误：SELECT COUNT(task_id) AS task_num  FROM task   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM task_questionnaire   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM  questionnaire_submitwhere questionnaire = '3',SELECT   COUNT(DISTINCT school_id) AS school_num FROM task_grade   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd'/n 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= '3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:09:45.523] [ERROR] logErr - 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= '3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:10:40.575] [ERROR] logErr - 执行sql发生错误：SELECT COUNT(task_id) AS task_num  FROM task   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM task_questionnaire   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT   COUNT(DISTINCT questionnaire_id) AS submit_questionnaire_num FROM  questionnaire_submitwhere questionnaire = '3',SELECT   COUNT(DISTINCT school_id) AS school_num FROM task_grade   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd'/n 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= '3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:10:40.579] [ERROR] logErr - 错误信息：Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '= '3'' at line 1
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:12:43.264] [ERROR] logErr - 执行sql发生错误：SELECT COUNT(task_id) AS task_num  FROM task   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM task_questionnaire   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd',SELECT  COUNT(questionnaire_id) AS submit_questionnaire_num FROM  questionnaire_submit where questionnaire = '3',SELECT   COUNT(DISTINCT school_id) AS school_num FROM task_grade   where project_id = '000e43a8-4fe5-2ffe-6146-2ac81bdff3fd'/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'questionnaire' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:12:43.270] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'questionnaire' in 'where clause'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:33:05.417] [ERROR] logErr - 执行sql发生错误：SELECT COUNT(task_id) AS task_num  FROM task   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM task_questionnaire   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b',SELECT  COUNT(service_id) AS submit_questionnaire_num FROM  questionnaire_submit where  service_id = 'sch_1',SELECT   COUNT(DISTINCT school_id) AS school_num FROM task_grade   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b'/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'service_id' in 'field list'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:33:05.423] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'service_id' in 'field list'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:33:43.502] [ERROR] logErr - 执行sql发生错误：SELECT COUNT(task_id) AS task_num  FROM task   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b',SELECT   COUNT(DISTINCT questionnaire_id) AS questionnaire_num FROM task_questionnaire   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b',SELECT  COUNT(service_id) AS submit_questionnaire_num FROM  questionnaire_submit where  server_id = 'sch_1',SELECT   COUNT(DISTINCT school_id) AS school_num FROM task_grade   where project_id = '43b71915-7e43-bec5-d928-d82da9a3809b'/n 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'service_id' in 'field list'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
[2017-05-22 15:33:43.508] [ERROR] logErr - 错误信息：Error: ER_BAD_FIELD_ERROR: Unknown column 'service_id' in 'field list'
    at Query.Sequence._packetToError (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (F:\work\QHPG-BJDF\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at PoolConnection.query (F:\work\QHPG-BJDF\node_modules\mysql\lib\Connection.js:208:25)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:40:13
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:718:13
    at iterate (F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:262:13)
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:274:29
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:44:16
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:723:17
    at F:\work\QHPG-BJDF\node_modules\mysql-queries\node_modules\async\lib\async.js:167:37
    at Query._callback (F:\work\QHPG-BJDF\node_modules\mysql-queries\lib\mysql-queries.js:42:7)
